VAMobile/src/utils/jsonFormatting.ts

```diff
@@ -0,0 +1,25 @@
+/**
+  Returns a string that replaces unicode/whitespace markup in json strings
+  \n Newline
+  \r Carriage Return
+  \u2028 Line Separator
+  \u2029 Paragraph Separator
+  \t Tab
+  \v Vertical Tab
+  \u2022 Bullet Points
+  @param text - array of textline to concatenate
+  */
+
+export const fixedWhiteSpaceString = (text: string | undefined) => {
```

Comment:
Do we expect this will be used in other places in the future? Just checking whether it needs to be in its own utils file, or can just be placed where it's used in the waygate stuff

Reply:
@alexandec this needs to be in it's own file as I have to ignore the file in the lint config. It doesn't allow double backslashes in the rules apparently even though that is the proper way to escape a back slash for regex.

Reply:
@Sparowhawk you can disable eslint for a single line, or for a block of code. Docs here: https://eslint.org/docs/latest/use/configure/rules#using-configuration-comments-1. I would recommend that rather than creating a separate file and adding it to the eslint config.

Reply:
no I cannot this particular eslint rule is not able to be disabled so the only solution I found was to ignore the file